.gitignore

```diff
@@ -56,3 +56,4 @@ gradlew.bat
 # GitHub Actions #
 ##################
 .github/.release
+src/test/resources/sdk-specification
```

Comment:
In `run_acceptance_tests.yml` you're making a link only to `features` but here it seems like you plan to link whole sdk-specification. It would be IMHO better if we would just make a decision and choose just one way